app/task-details/page.tsx

```diff
@@ -0,0 +1,9 @@
+import TaskDetails from "@/components/task-details";
+
+const TaskDetailsPage = () => {
+  return (
+    <TaskDetails />
```

Comment:
This is more like React single page application way of rendering everything on front-end. I suggest to use react server components (RSC) and server side rendering (SSR) for task details and task list pages, because they have mostly static content which can be pre-rendered on server side

Reply:
Okay, but I have one question. I need to make all components to be server-side too? As I understand, all my pages are already server-side. (If I remember correctly, if you don't write "use client", then by default the page will be server-side). Thanks!

Reply:
Some components will have to be client side because they use client side functionality (useState, useEffect, etc.). There is no rule when to have server components and when to have client components, but it will be good to have as much as possible on server side because it improves performance and decreases bundle size